index.html

```diff
@@ -30,33 +30,33 @@
         <div>
```

Comment:
- [ ] Although your links are looking great, We need to wrap everything inside a `ul` tag using a separate `li` tag.
https://github.com/brainskev/setup-and-mobile-version-skeleton/blob/7cd91a1185e09945d0ad9d85fbe02be92e70ef91/index.html#L151-L174